content/realtime/channel-lifecycle.textile

```diff
@@ -0,0 +1,141 @@
+
+h2(#metachannels). Metachannels
+
+Metachannels are a namespace of channels which all start with the @[meta]@ qualifier, uniquely identifying them from regular channels. An example of a metachannel would be @[meta]channel.lifecycle@.
+
+There are a number of metachannels available, which are:
+
+- log := This metachannel is used to broadcast log messages (usually error messages) for events that occur within the application's context
+- channel.lifecycle := This metachannel carries messages about channel lifecycle and metadata
+- connection.lifecycle := This metachannel carries messages about the lifecycle of realtime connections
+
+h2(#lifecycle-events). Lifecycle events
+
+The following events arise as messages on the channel lifecycle metachannel. In all cases, the @data@ member of the message is a "ChannelDetails":#channel-details instance.
+
+- channel.opened := indicates that the channel has been activated globally; that is, it has become active in at least one region, having previously been inactive. The included ChannelDetails.status includes "occupancy":#occupancy globally
+- channel.closed := indicates that the channel has been deactivated globally
+- channel.region.active := indicates that the channel has been activated in a specific region
+- channel.region.inactive := indicates that the channel has been deactivated in a specific region. The included @ChannelDetails.status@ includes "occupancy":#occupancy in that region
+- channel.occupancy := indicates a change in global "occupancy":#occupancy. Not all "occupancy":#occupancy changes are sent; there is a minimum interval on a per-channel basis, so if the "occupancy":#occupancy is changing continuously, then only periodic snapshots of occupancy are sent. Further roll-up of events may occur depending on the capacity of the lifecycle channel. Any "occupancy":#occupancy changes in which any of the "occupancy":#occupancy categories changes from 0 to a non-zero count, or vice versa, are always included
+
+It is worth noting that "occupancy":#occupancy is presently only available to "enterprise":https://www.ably.io/pricing/enterprise accounts and for testing. "Contact us":https://www.ably.io/contact to discuss your requirements if this is relevant to you.
+
+h3(#occupancy-realtime). Occupancy with realtime
+
+You can subscribe to channel "occupancy":#occupancy events by attaching to a meta channel using a realtime connection. However, realtime connections and channels have rate limits, typically "15 for free accounts":#https://support.ably.io/solution/articles/3000053845-do-you-have-any-connection-message-rate-or-other-limits-on-accounts. If the rate of channel lifecycle events exceeds this, a realtime connection is not going to be a reliable way to consume these events as you may get disconnected or lifecycle events may be discarded.
```

Comment:
Maybe specify the name of the metachannel exactly instead of saying 'a metachannel'?

Reply:
Maybe specify 15 per what i.e. 15/sec 15/month etc